src/app/App.jsx

```diff
@@ -1,71 +1,31 @@
 import React from 'react';
-import logo from '../logo.svg';
-import { Counter } from '../features/counter/Counter';
+import {
+  BrowserRouter as Router,
```

Comment:
That `as` looks weird. Do you have to do that?

Reply:
it's just a means of having cleaner code below. That way the router can be wrapped in a simple <Router></Router> tag. But I've removed the "as Router" from the import and now have the router wrapped in a <BrowserRouter></BrowserRouter> tag. Easy change, I think it just comes down to preference.